docs/developer-guide/modules-development-via-docker.md

```diff
@@ -1,71 +1,130 @@
+
+## How to install Docker for Windows
+
+* For Docker installation, first review the information at Docker for Windows: [What to know before you install](https://docs.docker.com/docker-for-windows/install/#what-to-know-before-you-install)
+* Install [Docker Desktop for Windows](https://docs.docker.com/docker-for-windows/install/) on your machine
+* During installation you'll need to choose Linux as operating system used inside your containers
 
-> You can parameterize these values ​​in the .env file
+## How to use
 
+1. Copy [ModulesDevelop](https://github.com/VirtoCommerce/vc-platform/blob/dev/DockerCompose/ModulesDevelop/) folder to local machine
+2. Create an external network for the Docker engine
+
+```cmd
+docker network create nat
 ```
+
+3. Parameterize values in the *.env* file.
+
+```cmd
 CMS_CONTENT_VOLUME=/Path/to/folder/cms-content
 APP_DATA_MODULES=/Path/to/folder/modules
 ```
-![image](../media/customize_mapping_folders.png)
 
-> You can also parameterize the platform version and search provider in the .env file
+You can also parameterize the platform version and search provider in the .env file
 
-```
-PLATFORM_VERSION=dev-3.10.0-alpha.1-bac031d3
+```cmd
+PLATFORM_VERSION=dev-linux-latest
 ...
 SEARCH_PROVIDER=ElasticSearch
 ```
 
-> The solution is launched and built using the command:
+4. Build and launch `docker-compose` by command:
 
-```
+```cmd
 docker-compose -f docker-compose.yml up --build -d
 ```
 
-## Installing and debugging the module
+5. Run Virto Commerce Platform Manager (configure modules an sample data)
+6. [Create new module](./create-new-module.md) as described in the article
+7. Write code for new module
+8. Build new module
+9. Connect new module with the platform
+10. Debug new module
+
+## Run Virto Commerce Platform Manager
+
+Once the containers are started, open VC Platform Manager - http://localhost:8090 . This will launch the application and install default modules. After modules have been installed you need to restart the container with the platform what gives you the opportunity to configure sample data.
 
-After launching the solution, you can install the necessary modules via the web interface of the platform. You need to restart the container with the platform. 
+![Choose sample data](../media/screen-sample-data.png)
 
-Now you can see that data for modules and applications has appeared in mapping folders.
+## Connect new module with the platform
 
-You need to remove the module from the installed ones via the platform.
+Use symbolic link to connect newly created solution folder to the Platform ~/Modules:
 
-![image](../media/remove_module.png)
-![image](../media/remove_module_2.png)
+* Run Command Prompt as an administrator.
+* Navigate to the physical location folder of `./Modules` folder on host machine.
+* Run the following command:
 
+```cmd
+mklink /d <your_module_name> <full_path_to_your_module_project>
+```
```

Comment:
Symbolic links don't seem to work with Docker for windows (running as linux environment). Can you double check? Also for the docker, should we tell users to switch to linux containers?)

Reply:
1. Yes, symbolic links don't.  Docs rewrite.
2. Added command to docker-compose for auto-restart container https://github.com/VirtoCommerce/vc-platform/blob/cee9b61126d1726c868fcc61d58977fd7d59fdc8/DockerCompose/ModulesDevelop/docker-compose.yml#L46